test/index.ts

```diff
@@ -111,3 +111,13 @@ test('Fails with PORT not in common range', async (t) => {
 	stdMock.restore()
 	stdMock.flush()
 })
+
+test('Works with prometheus', async (t) => {
```

Comment:
Could you add a test to check Prometheus metrics page ?

Reply:
like this one :
https://github.com/neo9/n9-node-routing/commit/76b892ecae72121985432adb108e79aa2b978ffa#diff-eddae6329159783f5cf0a069f102a188R22